.gitignore

```diff
@@ -0,0 +1,4 @@
+test
+.terraform
+terraform.tfstate
```

Comment:
I don't think we want to ignore the state file

Reply:
This is only so that I don't accidentally check in the state of my `./test` script.  It won't affect the consumers of the module.